docs/uml/protocol/InitializationPhase.puml

```diff
@@ -2,18 +2,18 @@
 MainController -> Listener: create()
 MainController -> SimScheduler: create()
 MainController -> Services: create()
-MainController -> SimScheduler: <font color=red>!</font> ScheduleTriggerMessage(\n\tInitializeServiceTrigger(0),\n\tweatherActorRef)
+MainController -> SimScheduler: <font color=red>!</font> ScheduleActivation(\n\tInitializeServiceTrigger(0),\n\tweatherActorRef)
```

Comment:
Triggers do not exist anymore. Have a look at the class `edu.ie3.simona.ontology.messages.SchedulerMessage.ScheduleActivation`: It works differently than `ScheduleTriggerMessage` from before.